# bioinformatics-notes
Collection of bioinformatics resources 

[R for Large Data & Bioinformatics](http://faculty.washington.edu/kenrice/bigr/)

[Intergrative Genome Biology](http://manuals.bioinformatics.ucr.edu)

- R Basics Manual
- BioConductor Manual
- NGS Analysis with R/Bioconductor
- NGS Analysis with Galaxy and IGV
- Programming in R
- Vim-R-Tmux: Command-Line IDE for R
- ChemmineR 
- EMBOSS
- Linux Essentials
- IIGB's Linux Cluster Manual

[WebMeV](http://mev.tm4.org/#/welcome)

WebMeV (Multiple Experiment Viewer) is a cloud-based application supporting analysis, visualization, and stratification of large genomic data, particularly for RNASeq and microarray data.

[Degust](https://degust.erc.monash.edu)

Degust is for RNA-seq exploration, analysis and visualisation

[Melbourne Bioinformatics](https://melbournebioinformatics.github.io/MelBioInf_docs/tutorials/rna_seq_dge_basic/rna_seq_basic_tutorial/)
RNA-Seq - Differential Gene Expression

[DIY.transcriptomics](http://diytranscriptomics.com/help)
A collection of local (UPenn), regional (Philly), and online places where you can go to learn more about using R/bioconductor.

[Draw Venn Diagram](http://bioinformatics.psb.ugent.be/webtools/Venn/)

[bioDBnet](https://biodbnet-abcc.ncifcrf.gov)

biological DataBase network is an application integrating a vast number of biological databases including Gene, UniProt, Ensembl, GO, Affy, RefSeq etc. The databases are created by downloading data from various public resources. They are formatted and maintained in a relational structure at the Advanced Biomedical Computing Center (ABCC).

[InteractiVenn](http://www.interactivenn.net)

[FGSEA on DESeq results](https://stephenturner.github.io/deseq-to-fgsea/)

[RNA-seq analysis in R](https://bioinformatics-core-shared-training.github.io/RNAseq-R/rna-seq-gene-set-testing.nb.html)

[Data Analysis in Genome Biology](https://girke.bioinformatics.ucr.edu/GEN242/)

GEN242 is a graduate class taught at the University of California, Riverside

[RNA-seq workflow: gene-level exploratory analysis and differential expression](http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html#annotating-and-exporting-results)

Here we walk through an end-to-end gene-level RNA-seq differential expression workflow using Bioconductor packages. We will start from the FASTQ files, show how these were quantified to the reference transcripts, and prepare gene-level count datasets for downstream analysis. We will perform exploratory data analysis (EDA) for quality assessment and to explore the relationship between samples, perform differential gene expression analysis, and visually explore the results.









